src/Async/Async.spec.js

```diff
@@ -150,7 +150,6 @@ test('Async fromPromise', t => {
   t.throws(fork(true), err, 'throws when true is returned from promise function')
   t.throws(fork([]), err, 'throws when an array is returned from promise function')
   t.throws(fork({}), err, 'throws when an object is returned from promise function')
-  t.throws(fork(unit), err, 'throws when an object is returned from promise function')
```

Comment:
HAHA. The spec text was wrong on this anyway!
Making life better, one commit at a time.